packages/monitor-v2/src/monitor-og/SnapshotVerification.ts

```diff
@@ -118,8 +211,8 @@ const getGraphqlData = async (
   });
 };
 
-// We don't want to throw an error if the IPFS request fails for any reason, so we return a stringified Error object
-// instead that will be logged by the bot.
+// We don't want to throw an error if the IPFS request fails for any reason, so we return an Error object instead that
+// will be logged by the bot.
```

Comment:
If you want to be as precise with types as possible (and you don't have to be), when you query this, it should be pulled down and put in an `unknown` type until you use your (very detailed) type guards to check if it's the type you expect.